### PR TITLE
Only display active APIs on explore page

### PR DESCRIPTION
**Pull request by imcgonigle**
### Description
This PR removes deprecated APIs from explore page

### CODE REVIEW

**Review by gileswells — DISMISSED** [review]
I don't think you _needed_ to update the variable name but looks good nonetheless.

File: src/containers/documentation/ExploreRoot.test.tsx [hunk 1]
```diff
@@ -39,6 +39,6 @@ describe('ExploreRoot', () => {
 
   it('should render mocked apis', () => {
     const apiCount = screen.getByTestId('api-count').textContent;
-    expect(apiCount).toBe(`${fakeAPIs.length}`);
+    expect(apiCount).toBe(7);
```
Comment (gileswells) [inline src/containers/documentation/ExploreRoot.test.tsx]: String formatting
```suggestion
    expect(apiCount).toBe('7');
```

**Review by gileswells — CHANGES_REQUESTED** [review]
Should be the last change

**Review by gileswells — DISMISSED** [review]
LGTM

**Review by andresriveratoro — APPROVED** [review]

**Review by andresriveratoro — APPROVED** [review]
LGTM

**Pull request merged**